src/ImageSharp/Formats/Webp/Lossless/LosslessUtils.cs

```diff
@@ -80,8 +80,10 @@ public static int FindMatchLength(ReadOnlySpan<uint> array1, ReadOnlySpan<uint>
         public static int VectorMismatch(ReadOnlySpan<uint> array1, ReadOnlySpan<uint> array2, int length)
```

Comment:
If array sizes are generally long enough - it can be easily vectorized.

Reply:
If you look at the commit history, i tried this: 491b742

It actually was twice as fast with benchmark.net and synthetic data, but with real data it did not perform well.
Usually the loop is quit early on, so it was actually slower then the non vectorized version.

Reply:
Hmmm, got it. Thanks for the explanation!